laws/src/main/scala/algebra/laws/Rules.scala

```diff
@@ -3,11 +3,17 @@ package laws
 
 import org.scalacheck.{Arbitrary, Prop}
 import org.scalacheck.Prop._
+import Prop.{False, Proof, Result}
 
 import algebra.std.boolean._
 
 object Rules {
 
+  def serializable[M](m: M): (String, Prop) =
+    "serializable" -> Prop { _ =>
+      Result(status = if (m.isInstanceOf[Serializable]) Proof else False)
```

Comment:
That's a good point. I'll try to rig up something with `DataOutputStream` and `DataInputStream` to test actual serialization. Does that seem reasonable?